composer.json

```diff
@@ -1,5 +1,5 @@
 {
-	"name": "ublaboo/datagrid",
+	"name": "peldax/datagrid",
```

Comment:
Could you please revert this? :)

Reply:
It is removed in the patch-2 branch as linked in my comment.